code/modules/admin/admin_verbs.dm

```diff
@@ -637,6 +639,30 @@ var/list/admin_verbs_mod = list(
 	message_admins("<span class='notice'>[ckey] creating an admin explosion at [epicenter.loc].</span>")
 	feedback_add_details("admin_verb","DB") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
+/client/proc/drop_emp()
+	set category = "Special Verbs"
+	set name = "Drop EMP"
+	set desc = "Cause an EMP of varying strength at your location."
+

```

Comment:
It's a proc, not a verb, admin verbs are designed so that they are only given to admins with the correct flags for them. Look, almost none of the procs in admin_verbs checks for rights.